src/splitInteger.test.js

```diff
@@ -8,12 +8,36 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger([10, 2]).constructor.name)
+      .toEqual('Array');
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(10, 1))
+      .toEqual([10]);
+  });
+
+  it(`should return parts elements`, () => {
+    expect(splitInteger(5, 5))
+      .toEqual([1, 1, 1, 1, 1]);
+  });
+
+  it(`should return an array length equal to parts`, () => {
+    expect(splitInteger(15, 5).length)
+      .toBe(5);
```

Comment:
You can also use [toHaveLength](https://jestjs.io/docs/expect#tohavelengthnumber) here